content/docs/notebooks/setup.md

```diff
@@ -0,0 +1,251 @@
+    
+
+  * **Custom**: If you select the custom option, you must specify a Docker image 
+    in  the form `registry/image:tag`. For guidelines on creating a Docker
+    image for your notebook, see the guide to 
+    [creating a custom Jupyter image](/docs/notebooks/custom-notebook/).
+
+1. Specify the total amount of **CPU** that your notebook server should reserve. 
+  The default is `0.5`. For CPU-intensive jobs, you can choose more than one CPU 
+  (for example, `1.5`).
+
+1. Specify the total amount of **memory** (RAM) that your notebook server should 
+  reserve. The default is `1.0Gi`.
+
+1. Specify a **workspace volume** to hold your personal workspace for this
+  notebook server. Kubeflow provisions a 
+  [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for your workspace volume. The PV ensures that you can
+  retain data even if you destroy your notebook server.
+
+  * The default is to create a new volume for your workspace with the
+    following configuration:
+  
+      * Name: `notebook-workspace`
+      * Size: `10Gi`
+      * Mount path: `/home/jovyan`
+      * Access mode: `ReadWriteOnce`
+
+  * Alternatively, you can point the notebook server at an existing volume by 
+    specifying the name, mount path, and access mode for the existing volume.
+
+1. *(Optional)* Specify one or more **data volumes** if you want to store and
+  access data from the notebooks on this notebook server. You can add new
+  volumes or specify existing volumes. Kubeflow provisions a 
+  [Kubernetes persistent volume (PV)](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) for each of your data volumes.
+
+1. Click **SPAWN** and wait a while. You should see an entry for your new
+  notebook server on the **Notebook Servers** page, with a spinning indicator in 
+  the **Status** column. It can take a few minutes to set up
+  the notebook server.
+
+    * You can check the status of your pod by entering the following command:
+
+        ```
+        kubectl -n <NAMESPACE> describe pods jupyter-<USERNAME>
+        ```
+
+        * Where `<NAMESPACE>` is the namespace you specified earlier 
+          (default `kubeflow`) and `<USERNAME>` is the name you used to log in.
+          **A note for GCP users:** If you have IAP turned on, the Pod has
+          a different name. For example, if you signed in as `USER@DOMAIN.EXT` 
+          the Pod has a name of the following form:
+
+          ```
+          jupyter-accounts-2egoogle-2ecom-3USER-40DOMAIN-2eEXT
+          ```
+
+1. When the notebook server provisioning is complete, you should see an entry
+  for your server on the **Notebook Servers** page, with a check mark in the
+  **Status** column:
+
+    <img src="/docs/images/notebook-servers.png" 
+      alt="Opening notebooks from the Kubeflow UI"
+      class="mt-3 mb-3 border border-info rounded">
+
+1. Click **CONNECT** to start the notebook server, and wait a while.
```

Comment:
Can we give users more guidance on how long they will need to wait? Or does this depend heavily on the environment that they are working in?

Reply:
I've removed the "wait" note here. It's more applicable when they click "spawn".